metadata-io/src/main/java/com/linkedin/metadata/graph/elastic/ElasticSearchGraphService.java

```diff
@@ -90,36 +93,25 @@ public void addEdge(@Nonnull final Edge edge) {
   }
 
   @Nonnull
-  public List<String> findRelatedUrns(
-      @Nullable final String sourceType,
-      @Nonnull final Filter sourceEntityFilter,
-      @Nullable final String destinationType,
-      @Nonnull final Filter destinationEntityFilter,
-      @Nonnull final List<String> relationshipTypes,
-      @Nonnull final RelationshipFilter relationshipFilter,
-      final int offset,
-      final int count) {
+  public List<String> findRelatedUrns(@Nullable final String sourceType, @Nonnull final Filter sourceEntityFilter,
```

Comment:
ughhhhh hate these reformattings

Reply:
this is much less readable imo. maybe we need to change the styling rules

Reply:
Me too :( I'm going to revert this file and add back the one function I added.